### ISSUE TITLE
Draw Image on pdf with existing image replace old image

### ISSUE THREAD

**Issue by julienGrd** [Hopding/pdf-lib#1082]
### What were you trying to do?

I try to edit a pdf document and let user draw on it :
- i load document with pdfJs (scale 1)
- i put a canvas on the same size over it where the user can draw
- on save, i extract my drawing canvas as png and use pdf-lib to draw the image over the PDF
=> just this work perfectly
- if i open again my pdf with previous drawing
- i put again a canvas where the user can draw
- when i save, the previous drawing is replaced by the new drawing zone
=> so the previous drawing is lost

### How did you attempt to do it?

//i show only the part when i use pdf-lib, which is the more relevant
````
const pages = this.pdfLib.getPages();
for (lCurrentPageNumber = 1; lCurrentPageNumber <= pages.length; lCurrentPageNumber++) {
            var lCurrentPage = pages[lCurrentPageNumber - 1];//attention commence à 0
             const { width, height } = lCurrentPage.getSize();
            var lContainer = document.getElementById("page-container-" + lCurrentPageNumber);
            var lDrawingCanvas = lContainer.querySelector('.canvas-container > .canvas-drawing');
             var dataURL = lDrawingCanvas.toDataURL("image/png");
            const pngImage = await this.pdfLib.embedPng(dataURL);
            lCurrentPage.drawImage(pngImage, {
                x: 0,
                y: 0, //attention on part du bas
                width: width,
                height: height,
            });
}
const pdfBytes = await this.pdfLib.save();
// pdfBytes have the current drawing but lost previous drawing zone which was embed
````

### What actually happened?

the pdf lost all previous image (drawing zone) embed in the doc

### What did you expect to happen?

i expect the pdf-lib draw image over it (its transparent png) or merge existing image into one, but not replace existing image

### How can we reproduce the issue?

- open a pdf, draw an image over it on the same size than the document (with transparency), and save it
- open the pdf previously saved, draw another image over it on the same size than the document (with transparency), and save it. The first image will be lost

### Version

1.17.0

### What environment are you running pdf-lib in?

Browser

### Checklist

- [X] My report includes a Short, Self Contained, Correct (Compilable) Example.
- [X] I have attached all PDFs, images, and other files needed to run my SSCCE.

**Comment by rafaelgomesxyz** [comment]
I think this is the same as #1075, which should be fixed in v1.17.1.

**Comment by julienGrd** [comment]
i confirm i don't reproduce the problem in v1.17.1, thanks !
